com.github.LongSoft.UEFITool.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.LongSoft.UEFITool
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+rename-desktop-file: uefitool.desktop
+rename-icon: uefitool
+command: uefitool
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
Looks like it uses the portals perfectly fine without `--filesystem=home`. Can you double-check if everything works for you, and if yes, drop it?

Reply:
Thank you for the review, `--filesystem=home` is indeed unnecessary and the application works fine without home access.